src/components/Hero_Page/Hero.js

```diff
@@ -1,130 +1,167 @@
 import React, { Component } from "react";
-import axios from "axios";
-
+import filteredData from "../Indexx/superFiltered.json";
 import "./Hero.scss";
 
 import CardIdentity from "./CardIdentity";
 import CardSuperId from "./CardSuperId";
 import CardPhysical from "./CardPhysical";
 import CardStats from "./CardStats";
 import CardAffiliation from "./Affiliation";
-import NavBar from '../NavBar/NavBar';
-import Footer from '../Footer/Footer';
-import Router from '../Router';
-import { Link } from 'react-router-dom';
-import Modal from '../Modal/Modal';
+import NavBarHero from "../NavBarHero/NavBarHero";
+import Footer from "../Footer/Footer";
+import { NavLink } from "react-router-dom";
+
+import ImgHeader from "./headerimg.jpg";
+
+// These variables will allow us to use a random button to change to another hero
+const randomId = Math.floor(Math.random() * filteredData.length);
+const randomHero = filteredData[randomId].id;
 
 export default class Hero extends Component {
   constructor(props) {
     super(props);
 
-    this.state = {
-      hero: []
-    };
-  }
+    //WIT THE NEXT FEW LINES WE FIND THE CURRENT HERO DISPLAYED ID IN THE FILTEREDDATA.JSON FILE
+    //WHICH WILL ALLOW US TO FIND WHICH ONE IS THE NEXT ONE OR THE PREVIOUS ONE
+    let currentfilteredDataID = 8;
+    // I CHOOSE A RANDOM VALID currentfilteredDataID JUST TO PREVENT AN ERROR PAGE IF THE CLIENT WRITE AN ID NUMBER INVALID IN THE URL
+    // FOR EXAMPLE THERE IS NO ID 16 AVAILABLE, BUT IF WE WRITE  http://localhost:3000/character/16 WE WILL SEE THANOS
+    filteredData.filter((element, idx) => {
+      if (element.id === this.props.match.params.id) {
+        return (currentfilteredDataID = idx);
+      }
+    });
```

Comment:
La même sans effet de bord : 
```suggestion
    currentfilteredDataID = filteredData.findIndex((element) => (element.id === this.props.match.params.id));
```